tests/test_cli.py

```diff
@@ -423,10 +423,15 @@ def test_cli_run_workflow(tmp_path, simple_df):
 
 
 @pytest.mark.parametrize(
-    "status, unit, exit_code", [("valid", "EJ/yr", 0), ("invalid", "EJ", 1)]
+    "status, unit, dimensions, exit_code",
+    [
+        ("valid_1", "EJ/yr", "region", 0),
```

Comment:
Maybe test for a list here?
```suggestion
        ("valid_1", "EJ/yr", ["region", "variable"], 0),
```